File: python/aprovados.py
N: int
media: float

N = int(input("Quantos alunos serao digitados? "))

nomes: [str] = [0 for x in range(N)]
notas1: [float] = [0 for x in range(N)]
notas2: [float] = [0 for x in range(N)]

for i in range(N):
    print(f"Digite nome, primeira e segunda nota do {i + 1}o aluno:")
    nomes[i] = str(input())
    notas1[i] = float(input())
    notas2[i] = float(input())

print("Alunos aprovados:")

for i in range(N):
    media = (notas1[i] + notas2[i]) / 2

    if media >= 6.0:
        print(nomes[i])

File: java/aprovados.java
import java.util.Locale;
import java.util.Scanner;

public class aprovados {

	public static void main(String[] args) {

		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);

		int N;
		double media;

		System.out.print("Quantos alunos serao digitados? ");
		N = sc.nextInt();

		String[] nomes = new String[N];
		double[] notas1 = new double[N];
		double[] notas2 = new double[N];

		for (int i = 0; i < N; i++) {
			System.out.printf("Digite nome, primeira e segunda nota do %do aluno:\n", i + 1);
			sc.nextLine();
			nomes[i] = sc.nextLine();
			notas1[i] = sc.nextDouble();
			notas2[i] = sc.nextDouble();
		}

		System.out.println("Alunos aprovados:");

		for (int i = 0; i < N; i++) {
			media = (notas1[i] + notas2[i]) / 2;

			if (media >= 6.0) {
				System.out.printf("%s\n", nomes[i]);
			}
		}

		sc.close();
	}
}